docs/development/security/gdpr_regulation.md

```diff
@@ -0,0 +1,156 @@
+
+
+### Legal rights of the data subject
+
+The GDPR regulation provides the data subject with an array of legal rights, related to the data and processing thereof. The data subject must be informed about:
+* The extent of the data collection
+* Transfer of data to any third-party and/or parties outside the EU
+Automated decision-making, based solely on algorithmic basis
+* Their privacy rights under GDPR, including right to
+    * revoke consent.
+    * access to view the data, and an overview.
+    * data portability, such as gaining access to a portable copy of the data, and in some cases to request that personal data be transferred from one data controller to another.
+    * be forgotten, that is, to have personal data deleted and erased, if conditions mentioned in the regulation is fulfilled.
+    * correction of wrong personal information.
+    * restrict the processing of personal data, if a number of conditions is met.
+    * contest being subject to automated decision-making, based solely on automated processing, such as profiling.
+    * file complaints with a Data Protection Authority (DPA) over processing of personal information, with the only exceptions to this right being that:
+        * Legal or official authority is being carried out
+        * ‘Legitimate interest’ where the organisation needs to process data in order to provide the data subject with a service they signed up for.
+        * A task being carried out for public interest.
+
+### Children specific regulations
+
+If the child is under the age of 16, it is only legal to collected and process data if a parent with custody of that child, gives the permission to do so. It falls under the responsibility of the data-responsible to ensure with reasonable effort that this is done. The individual countries might enact national regulation that puts this age down to 13 years.
+
+### Controller and Processor
+
+The data controller might be a company that seeks to collect and process personal information, while a data processor might be another company that provides a cloud-service that the data controller company wishes to use. 
+* Data protection principles and measures must be designed intro the business processes for products and services. This includes measures such as pseudonymisation and high level of privacy by default (Article 25). This is the responsibility of the data controller, even if processing is carried out by a data processor. The data controller keeps internal record of the processing of personal data. This will give an overview of the treatments of the data that is initiated. Such an overview is a necessary prerequisite to fulfill a number of obligations, such as: considerations on what information to process; handling insights requests and review of breaches of personal data security. This is an internal document that data protection agency can request at any time. 
```

Comment:
Just an informational comment. If you wonder why I didn't delete punctuations in this bulleted list, it is because they are full normal sentences by themself. As I understand it, that is what determines, whether or not you add punctuation in a bulleted list. :)

Reply:
Nice to know! Thanks! :)